Makefile

```diff
@@ -1,7 +1,13 @@
+# Makefile for ocr-fileformat
+
 PKG_NAME = ocr-fileformat
 PKG_VERSION = 0.2.3
 DOCKER_IMAGE = ubma/ocr-fileformat
 
+# Either get the version from Git (if available) or use PKG_VERSION.
+ROOTDIR = $(abspath $(dir $(MAKEFILE_LIST)))
+VERSION = $(shell test -d "$(ROOTDIR)/.git" && git -C "$(ROOTDIR)" describe --tags || echo $(PKG_VERSION))
```

Comment:
The `[` notation in combination with an `if` is IMO easier to read and we used it already in this Makefile. Should we change this line also into the other notation?

Reply:
I changed it now.